test/regress/regress0/parser/linear_arithmetic_err2.smt2

```diff
@@ -0,0 +1,10 @@
+; SCRUBBER: grep -o "Symbol 'mod' not declared as a variable"
+; EXPECT: Symbol 'mod' not declared as a variable
+; EXIT: 1
```

Comment:
Sorry, I actually believe that you need to add:

```
; REQUIRES: no-competition
```

for parse errors.

Reply:
Why do we need `no-competition` for parse errors?

Reply:
After resolving this question offline, I updated the tests where this line is required.